.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on:
+  push:
+    branches: [master]
```

Comment:
I think this should be any branch on `push` since we want CI to run in PRs before they are merged.

```suggestion
    branches: ['*']
```

Reply:
@mourner I've changed this up to run on any `push` event regardless of branch.